phoenix/core/settings.py

```diff
@@ -209,6 +209,8 @@
 NOTIFY_BEFORE_ETA = int(os.getenv('NOTIFY_BEFORE_ETA', '10'))
 
 DATADOG_TRACE = {
+    'AGENT_HOSTNAME': os.getenv('DATADOG_AGENT_HOSTNAME', 'localhost'),
+    'AGENT_PORT': os.getenv('DATADOG_AGENT_PORT', '8126'),
```

Comment:
Shouldn't this variable also be exposed in the compose file?

Reply:
yeah sorry only tested hostname and forgot to add that one